codegen/facelift/templates/IPCAdapter.template.h

```diff
@@ -74,10 +68,11 @@ class {{classExport}} {{interfaceName}}IPCAdapter: public ::facelift::IPCService
     using BaseType = ::facelift::IPCServiceAdapter<{{interfaceName}}>;
 
     {{interfaceName}}IPCAdapter(QObject* parent = nullptr);
+    ~{{interfaceName}}IPCAdapter();
 
-#ifdef DBUS_IPC_ENABLED
-    {{interfaceName}}IPCDBusAdapter m_ipcDBusServiceAdapter;
-#endif
+    struct Impl;
+
+    Impl* m_impl;
```

Comment:
can std::unique_ptr be used here instead?

Reply:
no, for some obscure reason, although it works on the IPCProxy side...